src/delete-pages/delete-pages.js

```diff
@@ -0,0 +1,29 @@
+import db from '../pouchdb'
+import { findPagesByUrl, updatePageSearchIndex } from '../search/find-pages'
+
+const defaultResultLimit = 30
+
+//Delete all Page entries returned by findPagesByUrl
+function deletePageEntries(pagesResult) {
+  console.log(pagesResult);
+  const pages = pagesResult.rows.map(row => ({
+    id: row.doc._id,
+    rev: row.doc_rev
+  }));
+  for (var key in pages) {
+    db.get(pages[key].id).then(doc => db.remove(doc._id, doc._rev)).then(() => {
```

Comment:
Why fiddle around with `id`s and `rev`s and get each page again? Just running `pagesResult.rows.map(row => db.remove(row.doc))` should work, no?